system/assets/js/admin/presidecore/formFields.js

```diff
@@ -7,14 +8,56 @@
 	$(".auto-slug").each( function(){
 		var $this = $(this)
 		  , $basedOn = $this.parents("form:first").find("[name='" + $this.data( 'basedOn' ) + "']");
+		 
+		 if($this.val() != ''){
+		 	URLslug($this.val(),ulrPrefix);
+		 }
 
 		$basedOn.keyup( function(e){
 			var slug = $basedOn.val().replace( /\W/g, "-" ).replace( /-+/g, "-" ).toLowerCase();
-
 			$this.val( slug );
+			URLslug(slug,ulrPrefix);
+		} );
+		$this.keyup( function(e){
+			var slug = $this.val().replace( /\W/g, "-" ).replace( /-+/g, "-" ).toLowerCase();
+			$this.val( slug );
+			URLslug(slug,ulrPrefix);
 		} );
 	});
 
+	function URLslug(slug,prefix){
```

Comment:
Similarly, this function needs to sit within the closure that starts on line 8 and refer to `$this`, rthat than `$(".auto-slug")`. Otherwise, you cannot have multiple auto slug fields in a single page.